poprox-db/migrations/versions/2024_09_30_1755-8418372f011f_create_demographics_table.py

```diff
@@ -0,0 +1,52 @@
+"""create_demographics_table
+
+Revision ID: 8418372f011f
+Revises: f7953f878964
+Create Date: 2024-09-30 17:55:31.059281
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '8418372f011f'
+down_revision: Union[str, None] = 'f7953f878964'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "demographics",
+        sa.Column(
+            "demographic_id",
+            sa.UUID,
+            primary_key=True,
+            server_default=sa.text("gen_random_uuid()"),
+        ),
+        sa.Column("account_id", sa.UUID, nullable=False),
+        sa.Column("gender", sa.String(50), nullable=False),
```

Comment:
I don't really think we're using fixed-length strings anywhere else in the codebase. At the same time I don't _mind_ having them, especially where we have good reason to think there is a valid upper-limit.

Reply:
Do you want me to roll that back? I didn't put much thought into it other than the fact that it felt right - not very sound reasoning.

Reply:
Nah, I think this is fine. since we have a fixed vocabulary for these.